src/_scss/pages/search/filters/programSource/programSource.scss

```diff
@@ -36,8 +36,47 @@
             @import "elements/filters/_typeahead";
             .program-source-select-filter__label {
                 @include display(flex);
-                @include justify-content(space-between);
+                @import '../../../../components/visualizations/tooltip/csstooltip';
```

Comment:
@ebdabbs,

not in scope for this pr but would like to discuss the performance effect of @import-ing scss like this multiple times in multiple places.

This @import is only used once, but there are many places we're using this pattern and importing the same piece of code multiple times which I'm afraid may be bloating our compiled css.